tests/ScheduledExecutionTest.php

```diff
@@ -117,7 +117,7 @@ public function testScheduledExecutionInterval()
         $job = $test->ScheduledJob();
         $job->execute();
 
-        $test = DataObject::get_by_id(TestScheduledDataObject::class, $test->ID);
+        $test = TestScheduledDataObject::get()->setUseCache(true)->byID($test->ID);
```

Comment:
Why are this using the cache, seems like in all the other tests you're explicitly not using the cache?

Reply:
Oops!
I did a lot of the changes with a find-and-replace, must have done this one in the wrong batch.